Cargo.toml

```diff
@@ -23,6 +23,7 @@ default = ["avx", "sse"]
 # On every other platform, these features do nothing, and RustFFT will behave like they are not set.
 avx = []
 sse = []
+neon = []
```

Comment:
I'm a little torn here, between naming this feature `neon` and naming it `neon-nightly`.

An argument for naming it `neon-nightly` is that it'll cover our ass if anything in our code stops compiling because of nightly. An argument against it, is that after neon support stabilizes, that feature will become vestigial but unremovable. What do you think?